redis/src/test/scala/zio/redis/ApiSpec.scala

```diff
@@ -20,6 +24,15 @@ object ApiSpec
     with HashSpec
     with StreamsSpec {
 
+  case class RedisExecutorWrapper(re: RedisExecutor.Service)
+
+  val secondConfigLayer = ZLayer.succeed(RedisConfig("localhost", 6380))
+
+  lazy val live: ZLayer[Logging with Has[RedisConfig], RedisError.IOError, Has[RedisExecutor.Service]] =
+    ZLayer.identity[Logging] ++ ByteStream.live >>> RedisExecutor.StreamedExecutor
```

Comment:
Can't you just pass a different configuration to the existing `RedisExecutor.live`?

Reply:
Maybe, will look into that